includes/apple-exporter/builders/class-components.php

```diff
@@ -461,7 +461,7 @@ private function get_anchor_buffer( $component ) {
 
 			// Add some buffer for the caption.
 			$ratio /= 1.2;
-		} elseif ( 'photo' === $component['role'] && ! empty( $component['URL'] ) ) {
+		} elseif ( ( 'photo' === $component['role'] || 'image' === $component['role'] ) && ! empty( $component['URL'] ) ) {
```

Comment:
Not exactly sure when/where this is applied but was grepping around for `'photo'` and figured it'd be a good idea to extend this styling/spacing logic to `'image'`s as well.

Reply:
Nice catch.